{
  "file_data": [
    {
      "desc": {
        "cause": "Under Investigation",
        "cluster": false,
        "cust_a": {
          "val": 11
        },
        "etr": "2020-11-20T21:00:00-0400",
        "start": "2020-11-20T13:55:12-0400"
      },
      "geom": {
        "p": [
          "{deqG~qu|J"
        ]
      },
      "id": "1",
      "title": "Outage Information"
    }
  ],
  "file_title": "030322"
}
